tests/unit/test_util_types.py

```diff
@@ -292,3 +292,5 @@ def test_version_compare(self):
         self.assertTrue(Version('4.0.0.build6-SNAPSHOT') > Version('4.0.0.build5-SNAPSHOT'))
         self.assertTrue(Version('4.0-SNAPSHOT2') > Version('4.0-SNAPSHOT1'))
         self.assertTrue(Version('4.0-SNAPSHOT2') > Version('4.0.0-SNAPSHOT1'))
+
+        self.assertTrue(Version('4.0.0-alpha1-SNAPSHOT') > Version('4.0.0-SNAPSHOT'))
```

Comment:
you should probably add Labs switch case here ;-)

Reply:
Also a reported problem info could be more comprehensive e.g. include problematic values and expected number.